plugins/catalog/src/alpha/plugin.tsx

```diff
@@ -45,6 +46,7 @@ export default createPlugin({
     viewTechDoc: convertLegacyRouteRef(viewTechDocRouteRef),
     createComponent: convertLegacyRouteRef(createComponentRouteRef),
     createFromTemplate: convertLegacyRouteRef(createFromTemplateRouteRef),
+    unregister: convertLegacyRouteRef(unregisterRedirectRouteRef),
```

Comment:
https://github.com/backstage/backstage/pull/21481/files#diff-8172f1eed894166ce8329a218913d4b4b6dc47a3357ec8e6d404b9774c4dabf6 😅, I'll extract it, https://github.com/backstage/backstage/pull/21495